test/services/mock_api.ts

```diff
@@ -0,0 +1,194 @@
+import deepEqual from "deep-equal";
```

Comment:
Done. I suppose this was a rebase issue: since the original commit modified this file (by adding an `export`) and then it was deleted, the result of a rebase is that the entire file was added.